### ISSUE TITLE
Generated code contains both tabs and spaces

### ISSUE THREAD

**Issue by vde-developer** [pdevito3/craftsman#87]
When using craftsman, generated classes contain a mix of tabs and spaces. Yes i know it doesn't matter functionally, and i don't have an opinion on tabs or spaces, just the mix of both can cause empty diffs in some diff software.

![image](https://user-images.githubusercontent.com/101636831/179550469-9ecf8b7f-2cdf-4181-8022-e33f1ce9a126.png)

Craftsman version:
0.15.0

**Comment by pdevito3** [comment]
Yeah there should be consistency here. Not sure when I'll get to this, but noted and I'll try to get to this. 

If you want to submit a PR to fix them or at least some of the higher traffic classes, I'll accept it.

In the meantime, I'd recommend a `dotnet format` to fix them after a scaffold and before a commit.

**Comment by vde-developer** [comment]
Thanks about the dotnet format tip!

I've checked the behavior of generating classes with Craftsman after replacing all tabs with 4 spaces, and it seems to work as for me... But i honestly haven't had enough experience using it to know if I'm missing anything. 

If anyone with more experience with Craftsman want to look I've created a pull request.
#88
